docs/API_Calls.md

```diff
@@ -0,0 +1,383 @@
+## Tranga API Calls
+This document serves to outline all of the different HTTP API calls that Tranga accepts. Tranga expects specific HTTP methods for its calls and therefore careful attention must be paid when making them. 
+In the examples below, `{apiUri}` refers to your `http(s)://TRANGA.FRONTEND.URI/api`. Parameters are included in the HTTP request URI and the request body is in JSON format. Tranga responses are always
+in the JSON format within the Response Body.
+
+#### [GET] /Connectors
+Retrieves the available manga sites (connectors) that Tranga is currently able to download manga from.
+
+- Parameters: 
+	None
+
+- Request Body: 
+	None
+	
+#### [GET] /Jobs
+Retrieves all jobs that Tranga is keeping track of, includes Running Jobs, Waiting Jobs, Manga Tracking (Monitoring) Jobs.
+
+- Parameters:
+	None
+	
+- Request Body: 
+	None
```

Comment:
Can have an optional body, to get a specific job
```
{
jobId: ${Tranga Job ID}
}
```
See https://github.com/C9Glax/tranga/blob/5f03b0d89cb15f410d49005fee7619d65524f265/Tranga/Server.cs#L172-L176